vars/wum-pipeline.groovy

```diff
@@ -0,0 +1,89 @@
+import groovy.json.*
+
+pipeline {
+    agent any
+    environment {
+        PATH = "/usr/local/Cellar/jq/1.5_3/bin:$PATH"
+        OS_USERNAME = credentials('OS_USERNAME')
+        OS_PASSWORD = credentials('OS_PASSWORD')
+        UAT_APPKEY = credentials('UAT_APPKEY')
+        LIVE_APPKEY = credentials('LIVE_APPKEY')
```

Comment:
These properties are not self explanatory.

Reply:
Done. Added comments.

Reply:
Shall we fix the property names itself?